src/pages/join.astro

```diff
@@ -70,6 +81,20 @@ const headings = [
         <a href="/join/slack" class="mb-2 me-3 btn btn-lg btn-dark" style="background-color: #611f69;">
             <i class="fab fa-slack"></i> Slack
         </a>
+        <a
+            href="https://www.linkedin.com/company/nf-core"
+            class="mb-2 me-3 btn btn-lg btn-dark"
+            style="background-color: #0077B5;"
+        >
+            <i class="fab fa-linkedin"></i> LinkedIn
+        </a>
+        <a
+            href="https://bsky.app/profile/nf-co.re"
+            class="mb-2 me-3 btn btn-lg btn-dark"
+            style="background-color: #1185fe;"
+        >
+            Bluesky
```

Comment:
Would be great to use the Bluesky logo, it's available on Iconify here: https://icon-sets.iconify.design/simple-icons/bluesky/

So should be able to use it with the `<Icon>` element, if added to `astro.config.mjs` as well.